Doc/library/dis.rst

```diff
@@ -258,10 +272,12 @@ operation is being performed, so the intermediate analysis object isn't useful:
    source line information (if any) is taken directly from the disassembled code
    object.
 
+   The *show_caches* and *adaptive* parameters function as in :func:`dis`.
```

Comment:
nit: I tripped up when reading this the first time because my mind went straight to function the noun.
```suggestion
   The *show_caches* and *adaptive* parameters work as they do in :func:`dis`.
```